geomesa-core/src/main/scala/org/locationtech/geomesa/core/index/STIdxStrategy.scala

```diff
@@ -70,11 +71,12 @@ class STIdxStrategy extends Strategy with Logging with IndexFilterHelpers {
   def buildSTIdxQueryPlan(query: Query,
                           iqp: QueryPlanner,
                           featureType: SimpleFeatureType,
+                          version: Int,
                           output: ExplainerOutputType) = {
-    val schema          = iqp.schema
+    val schema          = iqp.stSchema
     val featureEncoding = iqp.featureEncoding
-    val keyPlanner      = IndexSchema.buildKeyPlanner(iqp.schema)
-    val cfPlanner       = IndexSchema.buildColumnFamilyPlanner(iqp.schema)
+    val keyPlanner      = IndexSchema.buildKeyPlanner(schema)
+    val cfPlanner       = IndexSchema.buildColumnFamilyPlanner(schema)
```

Comment:
We could, but they only get used here I believe... this can be addressed as part of the ticket I opened for passing around materialized IndexSchema objects. I'd have to look closer to see exactly how often we parse them.